src/transformers/models/clip/modeling_clip.py

```diff
@@ -1334,6 +1411,7 @@ def __init__(self, config: CLIPConfig) -> None:
         super().__init__(config)
 
         self.num_labels = config.num_labels
+        config.vision_config._attn_implementation = config._attn_implementation
         self.vision_model = CLIPVisionTransformer(config.vision_config)
```

Comment:
Same here re setting / passing the attn_implementation

Reply:
https://github.com/huggingface/transformers/commit/89dab66fa51671f02363dec1613a9bae57f34d67